muesli/web/viewsLecture.py

```diff
@@ -450,7 +450,7 @@ def emailTutors(request):
         message = Message(subject=form['subject'],
```

Comment:
We can add a tag to the subject. How about this?

```suggestion
        message = Message(subject='[Tutoren] {}'.format(form['subject']),
```